qiskit/synthesis/__init__.py

```diff
@@ -77,6 +77,13 @@
 
    SolovayKitaevDecomposition
 
+Two Qubit Synthesis
+===================
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   SQiSWDecomposer
```

Comment:
Should it be called `SQiSWDecomposer` or `SQiSWDecomposition` like `SolovayKitaevDecomposition`?

Reply:
Right now it's named like TwoQubitBasisDecomposer and XXDecomposer which are more related to this class as they all take SU(4) and decompose it over some basis.
I agree we need consistency and my intent is to do this as I move stuff over to qiskit.synthesis (over a few different PRs)